app/scripts/metamask-controller.js

```diff
@@ -217,6 +217,9 @@ export default class MetamaskController extends EventEmitter {
       ),
       legacyAPIEndpoint: `${gasApiBaseUrl}/networks/<chain_id>/gasPrices`,
       EIP1559APIEndpoint: `${gasApiBaseUrl}/networks/<chain_id>/suggestedGasFees`,
+      determineNetworkStatusInfoUrlTemplate: ({ chainId }) => {
```

Comment:
The changes introduced in https://github.com/MetaMask/controllers/pull/609 require a new argument to be passed to GasFeeController to specify the new endpoint we hit to determine whether the network is busy.